src/components/status.jsx

```diff
@@ -693,9 +693,13 @@ function Status({
     x: 0,
     y: 0,
   });
-  const bindLongPress = useLongPress(
+  const bindLongPressContext = useLongPress(
     (e) => {
       const { clientX, clientY } = e.touches?.[0] || e;
+      // link detection copied from onContextMenu because here it works
+      const link = e.target.closest('a');
+      if (link && /^https?:\/\//.test(link.getAttribute('href'))) return;
+      e.preventDefault();
```

Comment:
and, as in onContextMenu, this prevents the default browser menu when NOT long-pressing a link, making sure the app context menu is the only one which opens.